package-lock.json

```diff
@@ -3204,7 +3204,7 @@
 		"arrify": {
 			"version": "1.0.1",
 			"resolved": "https://registry.npmjs.org/arrify/-/arrify-1.0.1.tgz",
-			"integrity": "sha512-3CYzex9M9FGQjCGMGyi6/31c8GJbgb0qGyrx5HWxPd0aCwh4cB2YjMb2Xf9UuoogrMrlO9cTqnB5rI5GHZTcUA==",
+			"integrity": "sha1-iYUI2iIm84DfkEcoRWhJwVAaSw0=",
```

Comment:
These hashes change format due to issues with the version of npm we use. To avoid this problem do the following:

- revert the package lock file to match the main branch.
- delete your local node_modules folder (it caches the hashes, and is the source of this problem)
- npm install again.